editor/plugins/sprite_frames_editor_plugin.h

```diff
@@ -38,6 +38,7 @@
 #include "scene/gui/dialogs.h"
 #include "scene/gui/item_list.h"
 #include "scene/gui/line_edit.h"
+#include "scene/gui/option_button.h"
```

Comment:
This can be forward-declared and moved to cpp.

Reply:
Will take a look tomorrow! Thank you